lib/github.js

```diff
@@ -22,18 +36,20 @@ module.exports = {
     return !!commit;
   },
 
-  createStatus: async (owner, repo, sha, target_url) => {
-    let github = new Octokit({
-      auth: GITHUB_TOKEN
-    });
-    return github.repos.createCommitStatus({
+  createStatus: async (owner, repo, head_sha, entries) => {
+    const github = await getRestClient();
+    return github.rest.checks.create({
       owner,
       repo,
-      sha,
-      state: 'success',
-      target_url,
-      description: 'Deployed to Incrementals.',
-      context: 'continuous-integration/jenkins/incrementals'
+      name: 'Incrementals',
```

Comment:
this also looks like a change. Your changing from a commit status to a check?

Reply:
Yup, so I can get multi modules giving all links, was horrid UX for jenkins core before and others you had to replace the first modules artifact id with the one you wanted in the URL, it never got the hpi / war

Reply:
I had filed an issue to this effect somewhere. Maybe only https://github.com/jenkins-infra/helpdesk/issues/2426.

Reply:
There we go, https://issues.jenkins.io/browse/JENKINS-52116 (CC @oleg-nenashev).